hello.template

```diff
@@ -37,10 +38,16 @@ def kill_server():
     bokeh_process2.kill()
 
 
-class IndexForm(FlaskForm):
+@app.route('/autocomplete', methods=['GET'])
+def autocomplete():
+    search = request.args.get('q')
     filelist = sorted(glob.glob(os.path.expanduser('~/data/*.ssw11.hhr')))
-    filename = SelectField('filename',
-                choices = [(os.path.basename(f).split('.')[0], os.path.basename(f).split('.')[0]) for f in filelist])
+    orflist = [os.path.basename(f).split('.')[0].upper() for f in filelist]
+    filteredlist = [orf for orf in orflist if orf.find(search.upper())>=0 ]
```

Comment:
You could also do `orf for orf in orflist if search in orf`

Reply:
Sweet... 😄